docroot/modules/custom/cs_generated_content/generated_content/node/civictheme_page_variations/03.general.inc

```diff
@@ -55,6 +55,35 @@ function _cs_generated_content_create_node_civictheme_page__variations__general(
       'space' => $helper::civicthemeSpaceTypeBoth(),
       'components' => _cs_generated_content_create_node_civictheme_page__variations_components_content_default(),
     ],
+    [
+      'title' => 'Demo Page, Summary, Topics (1), Thumbnail, Both spaces, Content, Show Last Updated Date, Not Custom Date',
+      'summary' => $helper::staticPlainParagraph(),
+      'topics' => $helper::staticTerms('civictheme_topics', 1),
+      'thumbnail' => $helper::staticMediaItem('civictheme_image'),
+      'space' => $helper::civicthemeSpaceTypeBoth(),
+      'show_last_updated' => TRUE,
+      'custom_updated_date' => $helper::randomDate('last month'),
```

Comment:
I did not tell you this - my fault, sorry - the variants need to have content that does not change with time. this is because we have visual regression for all components and it will be showing this field as "changed" every month we are doing a new test run.

Can you please update this to use a static date - 1 july 2022 for the first case and 14 of July  for the second case